recipes/heyoka.py/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+mkdir build
+cd build
+
+if [[ "$target_platform" == osx-* ]]; then
+    export AR_CMAKE_SETTING="-DCMAKE_CXX_COMPILER_AR=$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-ar -DCMAKE_C_COMPILER_AR=$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-ar"
+    export RANLIB_CMAKE_SETTING="-DCMAKE_CXX_COMPILER_RANLIB=$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-ranlib -DCMAKE_C_COMPILER_RANLIB=$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-ranlib"
```

Comment:
```suggestion
    export AR_CMAKE_SETTING="-DCMAKE_CXX_COMPILER_AR=$AR"
    export RANLIB_CMAKE_SETTING="-DCMAKE_CXX_COMPILER_RANLIB=$RANLIB -DCMAKE_C_COMPILER_RANLIB=$RANLIB"
```

Reply:
@isuruf I think I tried this at 

https://github.com/conda-forge/staged-recipes/pull/13961/commits/c161caa26904391a894aff8d569b413ede5e8b47

but it did not seem to work?

Reply:
Can you send a PR to add these to https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/master/recipe/activate-clang.sh#L131? I think that might be easier

Reply:
Gonna try again...